client/src/components/v2/CameraRecording.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Card } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import { emit } from 'api/v2/socket';
+import PropTypes from 'prop-types';
+import CameraRecordingStatus from './CameraRecordingStatus';
+
+/**
+ * @typedef {object} CameraRecordingProps
+ * @property {string} devices Comma separated values of devices
+ */
+
+/**
+ * Camera recording buttons for starting and stopping video recording.
+ *
+ * Starts/stops recording for both displays.
+ *
+ * This is a feature intended for V3 but is current in V2 for testing.
+ *
+ * @param {CameraRecordingProps} props Props
+ * @returns {React.Component<CameraRecordingProps>} Component
+ */
+export default function CameraRecording({ devices }) {
+  const startRecording = () => {
```

Comment:
These helper functions should be in a module in the api folder. The reason for this is that this allows us to provide a stable interface to the display components by abstracting away the implementation details of the API. For instance, V3 may replace "emit" with another method of communication.

Reply:
Done
480c632